packages/graphql/src/classes/utils/asserts-indexes-and-constraints.ts

```diff
@@ -272,18 +281,33 @@ async function getMissingConstraints({
 
     const missingConstraints: MissingConstraint[] = [];
 
-    nodes.forEach((node) => {
-        node.uniqueFields.forEach((field) => {
-            const property = field.dbPropertyNameUnescaped || field.fieldName;
-            if (node.getAllLabels().every((label) => !existingConstraints[label]?.includes(property))) {
+    for (const entity of schemaModel.concreteEntities) {
+        const entityAdapter = new ConcreteEntityAdapter(entity);
+        for (const uniqueField of entityAdapter.uniqueFields) {
+            if (!uniqueField.annotations.unique) {
+                continue;
+            }
+
+            let anyLabelHasConstraint = false;
+            for (const label of entity.labels) {
+                // If any of the constraints for the label already exist, skip to the next unique field
+                if (existingConstraints[label]?.includes(uniqueField.databaseName)) {
+                    anyLabelHasConstraint = true;
+                    break;
+                }
+            }
+            if (anyLabelHasConstraint === false) {
+                const constraintName =
+                    uniqueField.annotations.unique.constraintName || `${entity.name}_${uniqueField.databaseName}`;
+
                 missingConstraints.push({
-                    constraintName: field.unique!.constraintName,
-                    label: node.getMainLabel(),
-                    property,
+                    constraintName,
+                    label: entityAdapter.getMainLabel(),
```

Comment:
should we then also use the main label in the constraint name instead of `entity.name`?